dvc/commands/dag.py

```diff
@@ -28,6 +28,32 @@ def _show_dot(G):
     return dot_file.getvalue()
 
 
+def _show_mermaid(G, markdown: bool = False):
+    from dvc.repo.graph import get_pipelines
+
+    pipelines = get_pipelines(G)
+
+    graph = ""
+    if markdown:
+        graph += "```mermaid\n"
+    graph += "flowchart TD"
```

Comment:
If you go by the suggestion in https://github.com/iterative/dvc/pull/7383/files#r854739745.
```suggestion
    graph = "flowchart TD"
```